CMakeLists.txt

```diff
@@ -41,6 +41,9 @@ endif()
 
 cmake_minimum_required(VERSION 3.16)
 
+# globally add VKB_DEBUG for the debug build
+add_compile_definitions($<$<CONFIG:DEBUG>:VKB_DEBUG>)
```

Comment:
Global compile definitions are generally a bad practice. But this seems to be the simplest solution.

Did you try adding the compile definition in the `add_project` function?

Reply:
Cool, I am happy if others are. As discussed on slack, this works as the name of the define is so unique